aspen/http/__init__.py

```diff
@@ -0,0 +1,12 @@
+"""
+:mod:`http`
+===========
+
+Aspen doesn't implement all of HTTP, only things related to filesystem
+dispatch.
+
+.. automodule:: aspen.http.mapping
+.. automodule:: aspen.http.request
+.. automodule:: aspen.http.resource
+
+"""
```

Comment:
Actually, it looks like we can kill Querystring, so we could have an `aspen.urlpath` module. `Mapping` could maybe be in there, or in `aspen.mapping` (does Pando use the same `Mapping` class, or its own?).